src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -345,8 +346,9 @@ ApiGlobe.prototype.createSceneGlobe = function (coordCarto, viewerDiv) {
     this.scene = Scene(coordinate, ellipsoid, viewerDiv, debugMode, gLDebug);
 
     var map = new Globe(ellipsoid, gLDebug);
+    var updater = new GlobeUpdater(ellipsoid);
 
-    this.scene.add(map);
+    this.scene.setMap(map, updater);
```

Comment:
Why change `add` and use `setMap`, It should have no difference when you add a `node` or `globe` (`Globe` inherits `node`)